llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -54494,7 +54494,8 @@ static SDValue combineX86INT_TO_FP(SDNode *N, SelectionDAG &DAG,
 
 static SDValue combineCVTP2I_CVTTP2I(SDNode *N, SelectionDAG &DAG,
                                      TargetLowering::DAGCombinerInfo &DCI) {
-  bool IsStrict = N->isTargetStrictFPOpcode();
+  const SelectionDAGTargetInfo &TSI = DAG.getSelectionDAGInfo();
+  bool IsStrict = TSI.isTargetStrictFPOpcode(N->getOpcode());
```

Comment:
Worth adding a SelectionDAG::isTargetStrictFPOpcode helper wrapper to remove all the getSelectionDAGInfo calls in target code?

Reply:
I don't think it will make much of a difference, there are only four uses for now.
This is the same as querying InstrInfo / RegisterInfo: we could introduce wrappers for those, but they are so rarely used that no one cares.

Reply:
OK - no strong objection, it just looks messy tbh.